### DISCUSSION TITLE
How to edit Neorg prefix description in whick-key?

### DISCUSSION THREAD

**Discussion by weberam2** [LazyVim/LazyVim#2887] [Q&A]
Hello,

I would like to edit Neorg's whichkey description, but am not having any luck.

I have a whick-key.lua file in my plugins folder:

```
return {
  "folke/which-key.nvim",
  optional = true,
  opts = {
    defaults = {
      ["<leader>z"] = { name = "+snippets" },
      ["<localLeader>t"] = { name = "+toggle" },
    },
  },
}
```

but when I press my localleader it still just says prefix:
![image](https://github.com/LazyVim/LazyVim/assets/16805029/b7ed7862-d867-4a5d-a79b-2064280adeb0)

**Comment by dpetka2001** [comment]
There seems to be some kind of issue in `which-key` where it doesn't take into account the group names if there aren't any keys present and I believe `neorg` is only attaching the keys when in a `.norg` file. A workaround would be to just assign a dummy key along with the name. Something like 
```lua
  {
    "which-key.nvim",
    opts = {
      defaults = {
        ["<localleader>"] = {
          i = { name = "+insert", ["🚫"] = "which_key_ignore" },
          l = { name = "+list", ["🚫"] = "which_key_ignore" },
          m = { name = "+mode", ["🚫"] = "which_key_ignore" },
          n = { name = "+note", ["🚫"] = "which_key_ignore" },
          t = { name = "+mark", ["🚫"] = "which_key_ignore" },
        },
      },
    },
  },
```
Idea taken from https://github.com/folke/which-key.nvim/issues/514
